src/Uno.UI.RemoteControl.Host/Uno.UI.RemoteControl.Host.csproj

```diff
@@ -15,6 +15,7 @@
 	<ItemGroup>
 		<PackageReference Include="Microsoft.AspNetCore.App" />
 		<PackageReference Include="Mono.Options" Version="6.6.0.161" />
+		<PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
Do we really need this dependency ?

Reply:
Yes, it's been removed in .NET Core 3, and it's used internally by the RC server.